ffs/rpc/rpc.proto

```diff
@@ -3,6 +3,8 @@ package ffs.rpc;
 
 option go_package = "github.com/textileio/powergate/ffs/rpc";
 
+import "deals/rpc/rpc.proto";
```

Comment:
Bringing some work from a previous PR full circle. Can now import our other proto files and share message types. The generated go code imports now work properly